hca2scea-backend/test/golden/arguments.txt

```diff
@@ -0,0 +1,5 @@
+spreadsheet	HCA project uuid	E-HCAD accession	curator initials	technology	experiment type	factor values	public release date	hca last update date	study accession
```

Comment:
I would rename this file to tsv or csv, so it's rendered more nicely by github.

See https://docs.github.com/en/github/managing-files-in-a-repository/rendering-csv-and-tsv-data